<!DOCTYPE HTML>
<!--
	Phantom by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>DCJ - Data management</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
	</head>
	<body class="is-preload">
		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Header -->
					<header id="header">
						<div class="inner">

							<!-- Logo -->
								<a href="index.html" class="logo">
									<span class="symbol"><img src="images/logo.svg" alt="" /></span><span class="title">DC Johnson - Portfolio</span>
								</a>

							<!-- Nav -->
								<nav>
									<ul>
										<li><a href="#menu">Menu</a></li>
									</ul>
								</nav>

						</div>
					</header>

				<!-- Menu -->
					<nav id="menu">
						<h2>Menu</h2>
						<ul>
							<li><a href="index.html">Home</a></li>
							<li><a href="mangement.html">Managerial Experience</a></li>
							<li><a href="data.p1.html">Data management</a></li>
							<li><a href="publications.html">Publications</a></li>
							<li><a href="resume.html">C.Vs</a></li>
						</ul>
					</nav>

				<!-- Main -->
					<div id="main">
						<div class="inner">
							<h1>Data management</h1>
							<p>My publications have been analysed and summarised in R, Python and BASH with data from clinical trials databases and our in-house data server build on SQL. 
							Sequence analysis pipelines that I have utilised have been built and calling through python. I have been updating my old scripts to encompass newer techniques, 
							including utilising <a href="http://dreamchallenges.org/about-dream/">TCGA and</a><a href="https://www.kaggle.com/c/msk-redefining-cancer-treatment/"> DREAM 
							projects leaders approaches</a>. I have also been translating that to python more scalable machine and deep learning approaches 
							on my Github, that is currently private as contains patient data.</p>
							<p>LIMS experience included using Progeny LIMS for sample processing and bio-banking. I have also used LIMS together with liquid handlers automation including Hamilton (NGS Star) 
							and Qiagen (QIAglity) in conjunction with high-throughput assays.</p>
							<p>I'm experienced at dynamic documents, version control, containerization & workflow engines, I now work exclusively in JupterNotebook environments. In my last position 
							I worked on three informatic platforms/levels: On a desktop, with lots of (64GB) RAM running Rstudio and Ananconda; I maintained a local linux server for testing scripts 
							and housing shiny apps; utilised ICR Scientific Computing Infrastructure which included a 2,000 core, 2PB scratch High Performance Computing system, a 6.5 petabytes storage 
							and a 4x NVIDIA Tesla P100 GPU array. Over the last few months I have come to appreciate with huge sized sets that moving forward that most work will be carried out in Docker 
							containers and with interfaces such as Colab, Azure and Kaggle.</p>
							<p>I have extensive experience in analysing public genomic, transcriptomic and proteomics datasets, including:
							<br> <br> - NCBI GEO Datasets and EMBL-EBI ArrayExpress, through which I’ve regularly accessed .CEL Affymetrix U133+2 RNA arrays and SNP data for eQTL analysis, 
							prognostic signature work and hypotheses generating boxplots and prognostic curves (typically quantile or tertile), to reinforce molecular biology work within the lab, comparing preclinical experimental paired profiles, from 
							knockdown/upregulated, drug-treated pairs, using Limma/Deseq2 packages.
							<br><br> - NCBI GEO Datasets where I also accessed Cancer Cell Line Encyclopaedia (CCLE) data, which I used to examine characteristics of drug sensitive by IC50 to 
							examine to show similar patterns to which we showed in primary experiments.   
							<br><br> - European Genome-phenome Archive (EGA) for methylation data from blueprint consortium which I for replication and annotation; 
							<br><br> - Wash Epi-genome Browser for Chromatin Interaction Analysis by Paired-End Tag Sequencing ChIA-PET and ENCODE RoadMap browser again for annotation i.e. 
							Linc-RNA.
							<br><br> - MMRF Commpass study which is also part of TCGA and can be accessed through the GDC Data portal - International Myeloma consortium - To evaluate neutral 
							cancer status on overall and progression free survival: I utilised whole exon data, correcting variant allele frequencies (VAF) with copy number generated from low
							pass whole genome sequencing. In a recent DNA methylation biomarker project, I utilised pair-end RNA sequencing data (FPKM) in conjunction with FISH-seq data to carry
							out quartile cox proportional with age, sex, subgroup to replicate DNA methylation findings.
							</p>
							◦	Interpretation and presentation of multi-centre phase III clinical trial data including adaptive and maintenance arms. <br>
							◦	Meta-analysis of trial-based clinical end point data and combining data from multi-country real world datasets or cancer registries. <br>
							◦	Integration of translational data with clinical endpoints.<br>
							◦	Statistical analyses in SPSS, R and Python. <br>
							◦	Generation of publication ready clinical trials analyses for JCO, Nature Comms, Leukemia.<br>
							◦	Application of relevant statistical techniques include sample size/power calculation, cox regression, multi-variable analysis, ROC modelling, landmark analysis, C-stats and Bayesian methods. <br>
							◦	Collection of clinical trial data, data cleaning, HTA. <br>
							◦	Development of trials protocols, and REC approvals. <br>
							◦	Strong Microsoft Office skills (Word, Excel, PowerPoint)<br>
							◦	Image editing tools and processes utilizing GIMP2 and photoshop. <br>
							◦	Project tracking and management with MS project, Podio and Freedcamp. <br>
							◦	Registered Azure and AWS user.<br>
							◦	Collaborative working through Slack, Asana and Git/Jupyter notebooks.<br>
							◦	Keen twitter follower of health care luminaires and advocates<br>
							◦	Utilise bioRxiv to follow emerging research and technologies.  <br>  
							◦	I use messageboard resources including:https://www.rna-seqblog.com, https://www.reddit.com/r/bioinformatics/, https://www.feedspot.com/infiniterss.php?q=https%3A%2F%2Fwww.nature.com%2Fsubjects%2Fcomputational-biology-and-bioinformatics.rss<br>
						
						
						</div>
					</div>

				<!-- Footer -->
					<footer id="footer">
						<div class="inner">
							<section>
								<h2>Get in touch</h2>
								<form method="post" action="#">
									<div class="fields">
										<div class="field half">
											<input type="text" name="name" id="name" placeholder="Name" />
										</div>
										<div class="field half">
											<input type="email" name="email" id="email" placeholder="Email" />
										</div>
										<div class="field">
											<textarea name="message" id="message" placeholder="Message"></textarea>
										</div>
									</div>
									<ul class="actions">
										<li><input type="submit" value="Send" class="primary" /></li>
									</ul>
								</form>
							</section>
							<section>
								<h2>Follow</h2>
								<ul class="icons">
									<li><a href="#" class="icon brands style2 fa-twitter"><span class="label">Twitter</span></a></li>
									<li><a href="#" class="icon brands style2 fa-facebook-f"><span class="label">Facebook</span></a></li>
									<li><a href="#" class="icon brands style2 fa-instagram"><span class="label">Instagram</span></a></li>
									<li><a href="#" class="icon brands style2 fa-dribbble"><span class="label">Dribbble</span></a></li>
									<li><a href="#" class="icon brands style2 fa-github"><span class="label">GitHub</span></a></li>
									<li><a href="#" class="icon brands style2 fa-500px"><span class="label">500px</span></a></li>
									<li><a href="#" class="icon solid style2 fa-phone"><span class="label">Phone</span></a></li>
									<li><a href="#" class="icon solid style2 fa-envelope"><span class="label">Email</span></a></li>
								</ul>
							</section>
							<ul class="copyright">
								<li>&copy; Untitled. All rights reserved</li><li>Design: <a href="http://html5up.net">HTML5 UP</a></li>
							</ul>
						</div>
					</footer>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

			<div>Icons made by <a href="https://www.flaticon.com/authors/eucalyp" title="Eucalyp">Eucalyp</a> from <a href="https://www.flaticon.com/"             title="Flaticon">www.flaticon.com</a> is licensed by <a href="http://creativecommons.org/licenses/by/3.0/"             title="Creative Commons BY 3.0" target="_blank">CC 3.0 BY</a></div>
	</body>
</html>