<!DOCTYPE HTML>
<!--
	Phantom by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Admixture Genetics</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
	</head>
	<body class="is-preload">
		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Header -->
					<header id="header">
						<div class="inner">

							<!-- Logo -->
								<a href="index.html" class="logo">
									<span class="symbol"><img src="images/logo.svg" alt="" /></span><span class="title">DC Johnson - Portfolio</span>
								</a>

							<!-- Nav -->
								<nav>
									<ul>
										<li><a href="#menu">Menu</a></li>
									</ul>
								</nav>

						</div>
					</header>

				<!-- Menu -->
					<nav id="menu">
						<h2>Menu</h2>
						<ul>
							<li><a href="index.html">Home</a></li>
							<li><a href="mangement.html">Managerial Experience</a></li>
							<li><a href="data.p1.html">Data management</a></li>
							<li><a href="publications.html">Publications</a></li>
							<li><a href="elements.html">C.Vs</a></li>
						</ul>
					</nav>

				<!-- Main -->
					<div id="main">
						<div class="inner">
							<h1>Admixture Genetics</h1>
							<p><b>Background</b><br>
							Admixture genetics has been of great personal interest to me as my family and I are very heterogeneous in terms of geographical ancestry and I wonder how the great breakthroughs
							in genetics will translate to families such as mine. <br> <br>
							Admixture studies are important because:<br>
							◦ Healthcare is important for all ancestral populations and therefore is essential we understand impact of treatments in individuals spanning all population groups. In the gnomAD exome and
							genome database includes over 60% European sequences and less than 10% sequences from individuals of African ancestry at present. <br>
							◦ To accurately model genes and the influence on transcription and translation, it is important that we include as much information and variation as possible. Rare variant in one 
							population may be more common other populations, integrating eQTL and meQTL loci across populations for instance, can help build up phenotypic models. <br> 
							◦ Ancestral populations differ due to differing selection pressure on populations in response to environmental pressure, these variates that differ between populations are also likely to be 
							enriched in causal variation.<br><br> Left is an image from a Harvard review indicating the nature of population genetics, the center is a diagram from a cell review highlighting the disparity 
							in ancestry distribution in GWAS studies, right is a cover from the book of Blindspot: Hidden Biases of Good People. Given the disparity in the research community, we must continually 
							question the questions we are asking.
							</p>
							<span class="image main"><img src="images/admix.fig1.png" alt="" /></span>
							<p><b>GWAS population outliers</b><br>
							All of my GWAS studies have been confined to european populations. In order to avoid inflating association test results due to an enrichment of non-european individuals in either of the comparator 
							groups, it has been necessary for me to identify non-europeans cases and excluded them from the analysis. Principle component analysis (PCA) analysis together with the 
							<a href="http://reich.hms.harvard.edu/software"> EIGENSTRAT correction method,</a> allows us to identify population outliers and control for any inflation of test statistics by population 
							stratification using calculated eigenvectors.       
							</p>
							<span class="image main"><img src="images/admix.fig2.png" alt="" /></span>
							<p><b>Ancestry variation and causal variates</b><br>
							A variant linked with the somatic cancer translocation t11;14 risk in myeloma is more prevalent in African populations. This variant is a well-described splice variant in
							with results in short isoform of CCND1 which has an absent nucleus transporting motif, which leads to constitutive activation and increased proliferation. On the completion of our
							study we internally speculated that population disparity of the variant may lead to an increase of t11;14 translocation in myeloma patients of African descent. Some support for 
							this hypothesis was provided in a <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=30305608"> 2018 study.</a> 
						</p>
							<span class="image main"><img src="images/admix.fig3.png" alt="" /></span>
							<p><b>Programs and packages</b><br>
								Association testing - PLINK versions 1.7-1.9, IMPUTE, SNPTEST, EIGENSTRAT, Hail; Meta-analysis - PLINK, META, METAL; Mendelinan randomisation - custom R script, MendelianRandomization v0.3.0, MeRP; 
								Data-visualisation - ggplot2, SNAP, circlize, matplotlib, seaborn,; HLA region analysis - SNP2HLA; Hi-C- HICUP and CHiCAGO; eQTL and meQTL - matrixQTL, FastQTL and PEER.</p>
								<p><b>References</b><br>
							◦ Genome-wide association study identifies variation at 6q25.1 associated with survival in multiple myeloma. Johnson DC, Weinhold N, Mitchell JS, Chen B, Kaiser M, Begum DB, Hillengass J, Bertsch U, Gregory WA, Cairns D, Jackson GH, Försti A, Nickel J, Hoffmann P, Nöethen MM, Stephens OW, Barlogie B, Davis FE, Hemminki K, Goldschmidt H, Houlston RS, Morgan GJ. Nat Commun. 2016 Jan 8;7:10290. <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26743840"> PMID: 26743840 </a> <br><br>
								◦ Greenwald, A. G. & Banaji, M. R. (2013). Blindspot: Hidden Biases of Good People. Delacorte Press, ISBN 978-0553804645
								◦ Trans-ethnic association study of blood pressure determinants in over 750,000 individuals. Giri A, Hellwege JN, Keaton JM, Park J, Qiu C, Warren HR et al. Nat Genet. 2019  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=30578418"> PMID: 30578418 . </a>
								◦ The Missing Diversity in Human Genetic Studies.Sirugo, G., Williams, S. M., & Tishkoff, S. A. (2019). Cell, 177(1), 26–31. doi:10.1016/j.cell.2019.02.048 <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=30901543"> PMID: 30901543. </a>
								◦ Reply to 'Neutral tumor evolution?' Heide T, Zapata L, Williams MJ, Werner B, Caravagna G, Barnes CP. Nature Genetics 2018, Dec;50(12):1633-1637. <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=30374073"> PMID: 30374073. </a> <br><br></p><p>
						</div>
					</div>

				<!-- Footer -->
					<footer id="footer">
						<div class="inner">
							<section>
								<h2>Get in touch</h2>
								<form method="post" action="#">
									<div class="fields">
										<div class="field half">
											<input type="text" name="name" id="name" placeholder="Name" />
										</div>
										<div class="field half">
											<input type="email" name="email" id="email" placeholder="Email" />
										</div>
										<div class="field">
											<textarea name="message" id="message" placeholder="Message"></textarea>
										</div>
									</div>
									<ul class="actions">
										<li><input type="submit" value="Send" class="primary" /></li>
									</ul>
								</form>
							</section>
							<section>
								<h2>Follow</h2>
								<ul class="icons">
										<li><a href="@davecjohnson" class="icon brands style2 fa-twitter"><span class="label">Twitter</span></a></li>
										<li><a href="https://github.com/dcjohnson23" class="icon brands style2 fa-github"><span class="label">GitHub</span></a></li>
										<li><a href="https://scholar.google.co.uk/citations?user=yidb_JkAAAAJ&hl=en" class="icon brands style2 fa-google"><span class="label">Google</span></a></li>
										<li><a href="https://linkedin.com/in/david-johnson-ab90033b/" class="icon brands style2 fa-linkedin"><span class="label">Linkedin</span></a></li>
										<li><a href="davcj23@gmail.com" class="icon solid style2 fa-envelope"><span class="label">Email</span></a></li>
								</ul>
							</section>
							<ul class="copyright">
								<li>&copy; Untitled. All rights reserved</li><li>Design: <a href="http://html5up.net">HTML5 UP</a></li>
							</ul>
						</div>
					</footer>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

			<div>Icons made by <a href="https://www.flaticon.com/authors/eucalyp" title="Eucalyp">Eucalyp</a> from <a href="https://www.flaticon.com/"             title="Flaticon">www.flaticon.com</a> is licensed by <a href="http://creativecommons.org/licenses/by/3.0/"             title="Creative Commons BY 3.0" target="_blank">CC 3.0 BY</a></div>
	</body>
</html>